tests/contrib/mysql/test_mysql.py

```diff
@@ -58,7 +58,7 @@ def test_simple_query(self):
             },
         )
 
-    def test_simple_query_fetchll(self):
+    def test_simple_query_fetchall(self):
```

Comment:
Let's actually do this change in a separate PR. It seems silly to have to go through all that effort but isolating the changes makes it easier to rollback or cherry-pick in the event that we need to. (ex. if we end up changing something in mysql then we don't want to get merge conflicts because of this mariadb PR)

Reply:
Makes sense!